tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -253,6 +276,40 @@ const getContactInfoName = async () => {
   return (await contactInfoName()).getText();
 };
 
+const getContactMedicID = async () => {
+  await contactMedicID().waitForDisplayed();
+  return (await contactMedicID()).getText();
+};
+
+/**
+ * Add health programs to a health facility, standard config
+ * 
+ * @param {string} program 
+ * Refers to the different Health Programs, "anc" for Antenatal care,  "pnc" for Postnatal care,  "imm" for Immunizations and "gpm" for Growth monitoring (nutrition)
+ */
+const addHealthPrograms = async (program = 'anc') => {
+  await (await editContactButton()).waitForDisplayed();
+  await (await editContactButton()).click();
+  await healthProgram(program).click();
+  if(program === 'imm'){
```

Comment:
```suggestion
  if (program === 'imm') {
```

Reply:
You could have a constant for these programs. Usually constants are uppercase on the top of the file.
```
const HEALTH_PROGRAMS = { IMM: 'imm', ANC: 'anc' };
```
If you need the label as well, then:
```
const HEALTH_PROGRAMS = { 
   IMM: {
      code: 'imm',
      label: 'Immunizations'
   },
};
```